README.md

```diff
@@ -133,6 +133,151 @@ More usage examples:
 
 **Pull Requests are welcome!**
 
+#### Proof of Work (NIP-13)
+
+Proof of Work in Nostr allows clients to demonstrate computational effort spent on creating an event. This can be used for spam reduction by requiring events to have a minimum difficulty level.
+
+##### Basic Usage
+
+```csharp
+// Create an event
+var ev = new NostrEvent
+{
+    Kind = NostrKind.ShortTextNote,
+    CreatedAt = DateTime.UtcNow,
+    Content = "This message includes proof of work to demonstrate NIP-13"
+};
+
+// Mine the event with difficulty 16 (leading zero bits)
+var minedEvent = await ev.GeneratePow(16);
+
+// Once mined, sign it with your private key
+var key = NostrPrivateKey.FromBech32("nsec1xxx");
+var signedMinedEvent = minedEvent.Sign(key);
+
+// Check the actual difficulty achieved
+int difficulty = signedMinedEvent.GetDifficulty();
+Console.WriteLine($"Event ID: {signedMinedEvent.Id}");
+Console.WriteLine($"Difficulty achieved: {difficulty} bits");
+
+// Send to relay
+client.Send(new NostrEventRequest(signedMinedEvent));
+```
+
+##### Validating Proof of Work
+
+When receiving an event, you can verify if it meets a minimum difficulty requirement:
+
+```csharp
+// Check if an event has a valid PoW with a difficulty of at least 15 bits
+int minDifficulty = 15;
+bool isValid = receivedEvent.HasValidPow(minDifficulty);
+if (isValid)
+{
+    Console.WriteLine($"Event has valid PoW with difficulty: {receivedEvent.GetDifficulty()} bits");
+}
+else
+{
+    Console.WriteLine("Event does not have sufficient proof of work");
+}
+```
+
+##### Advanced: Using Delegate-based Mining for Progress Updates
+
+For longer mining operations, you may want to receive progress updates and have more control over the mining process. The library provides a delegate-based approach for this:
```

Comment:
I don't think this is mining. Is there a definition in the NIP?

Reply:
The term "mining" in this context refers to performing Proof of Work (PoW) as described in [NIP-13](https://github.com/nostr-protocol/nips/blob/master/13.md). The process involves incrementing a `nonce` value until the generated note ID meets a specific difficulty target (i.e., a required number of leading zero bits). This is commonly referred to as "mining" in other PoW-based systems as well.

If there’s a more precise term that aligns better with the Nostr ecosystem, I’d be happy to adjust the wording. However, the concept itself is directly derived from NIP-13.